shell/platform/tizen/accessibility_settings.cc

```diff
@@ -18,7 +14,23 @@
 namespace flutter {
 
 AccessibilitySettings::AccessibilitySettings(FlutterTizenEngine* engine)
-    : engine_(engine) {
+    : engine_(engine), screen_reader_enabled_(false) {
```

Comment:
I modified `screen_reader_enabled_` initialization in [88fcd99](https://github.com/flutter-tizen/engine/pull/254/commits/88fcd99a7db6e527347630d62511a627edfd2c7a).
This variable is needed, when I enable tts on mobile simulator, the `OnScreenReaderStateChanged` may be called multiple times sometimes, so I add this variable to prevent `SetSemanticsEnabled` repeatedly.